test/integration/developer_portal/admin/account/stripe_controller_test.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module DeveloperPortal
+  class Admin::Account::StripeControllerTest < ActionDispatch::IntegrationTest
+    include System::UrlHelpers.cms_url_helpers
+
+    def setup
+      @provider = FactoryBot.create(:provider_account, payment_gateway_type: :stripe, payment_gateway_options: {login: 'sk_test_4eC39HqLyjWDarjtT1zdp7dc', publishable_key: 'pk_test_TYooMQauvdEDq54NiTphI7jx'})
```

Comment:
Are these some bogus values?

Reply:
Exactly. All the values of stripe elements have been copied&pasted from Stripe Docs examples 😄